### PR TITLE
Cosmos 44

### PR DESCRIPTION
**Pull request by mikluke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,112 +1,71 @@
-# Decentr
-![go version](https://img.shields.io/github/go-mod/go-version/Decentr-net/decentr?color=blue)
```
Comment (megaherz) [inline README.md:2]: Do wee need to remove this?

File: README.md [hunk 2]
```diff
@@ -1,112 +1,71 @@
...
-cd decentr
-# Compile and install Decentr
-make install
-# Initialize decentrd in ~/.decentrd and name your node
-decentrd init yournodenamehere
-```
+To launch your blockchain live on multiple nodes, use `starport network` commands. Learn more about [Starport Network](https://github.com/tendermint/spn).
 
-Add Seeds:
+### Web Frontend
```
Comment (megaherz) [inline README.md:22]: Vue no needed

File: README.md [hunk 3]
```diff
@@ -1,112 +1,71 @@
...
-Download Genesis, Start your Node, Check your Node Status:
-
-```bash
-# Download genesis.json
-wget -O $HOME/.decentrd/config/genesis.json https://raw.githubusercontent.com/Decentr-net/testnets/master/1.4.7/genesis.json
-# Start Decentrd
-decentrd start
-# Check your node's status with decentrcli
-decentrcli status
+curl https://get.starport.network/Decentr-net/decentr@latest! | sudo bash
```
Comment (megaherz) [inline README.md:61]: Will it work?

File: README.md [hunk 4]
```diff
@@ -1,112 +1,71 @@
...
 ```
+`Decentr-net/decentr` should match the `username` and `repo_name` of the Github repository to which the source code was pushed. Learn more about [the install process](https://github.com/allinbits/starport-installer).
 
-Welcome to the Decentr Testnet!
+## Learn more
 
-## Follow us!
```
Comment (megaherz) [inline README.md:108]: This block is needed

File: app/app.go [hunk 5]
```diff
@@ -1,501 +1,624 @@
...
-		auth.FeeCollectorName:     nil,
-		distr.ModuleName:          nil,
-		mint.ModuleName:           {supply.Minter, supply.Burner},
-		staking.BondedPoolName:    {supply.Burner, supply.Staking},
-		staking.NotBondedPoolName: {supply.Burner, supply.Staking},
-		gov.ModuleName:            {supply.Burner},
-		operations.ModuleName:     {supply.Minter, supply.Burner},
+		authtypes.FeeCollectorName:     nil,
+		distrtypes.ModuleName:          nil,
+		minttypes.ModuleName:           {authtypes.Minter},
```
Comment (megaherz) [inline app/app.go:151]: minttypes.ModuleName is only Minter now, is it correct?

File: init.sh [hunk 6]
```diff
@@ -1,31 +0,0 @@
-rm -rf ~/.decentrd
```
Comment (megaherz) [inline init.sh:1]: We used this script for a quick node run. What should be used instead?

File: app/app.go (hunk 5 above)
Comment (mikluke) [reply app/app.go:151]: yep

File: README.md [hunk 7]
```diff
@@ -89,14 +88,14 @@ nano ~/.decentrd/config/config.toml
 Scroll down to `seeds` in `config.toml`, and replace with
 
 ```
-seeds = "36d036d0fe2d3c95950f77abdf4ff53a732f38e3@ares.testnet.decentr.xyz:26656,fcc45b026e948d7c81f46fdb650871c8bdc1378a@hera.testnet.decentr.xyz:26656,744449aff5e8797c9c403c56d0f5e6d2be52604b@hermes.testnet.decentr.xyz:26656,e5b6b3ba0bca1ea8427911843df24568feb53afc@poseidon.testnet.decentr.xyz:26656,a4771cec2e881ead305c68c1b3f8de7403786944@zeus.testnet.decentr.xyz:26656"
+seeds = "95a70f0119af52e54697fa7feb8b09b4e7c7ec21@ares.testnet.decentr.xyz:26656,b6d499b2b0146627b9bf6f33a9a7e4013312c6d1@hera.testnet.decentr.xyz:26656,576d044b24cc449366850a95f7616f03ab8d14b3@hermes.testnet.decentr.xyz:26656,c98511455134b4450ebb20fce57308a9fb300b89@poseidon.testnet.decentr.xyz:26656,acc5524b4ff34591357a28d5fccf4efb5ad883c5@zeus.testnet.decentr.xyz:26656"
```
Comment (megaherz) [inline README.md:91]: Dummy? Is it going to be updated as soon as the new Testnet is up?

File: .golangci.yml [hunk 8]
```diff
@@ -4,7 +4,7 @@ linters:
     - deadcode
     - depguard
     - dogsled
-    - dupl
+    # - dupl
```
Comment (megaherz) [inline .golangci.yml:7]: Is it an updated .golangci.yml for cosmos 44 or did you comment them yourself?

File: README.md [hunk 9]
```diff
@@ -20,7 +20,6 @@ This assumes that you're running Linux or MacOS and have installed [Go 1.15+](ht
 If you already have a previous version of Decentr installed:
 ```
 rm -rf ~/.decentrd
```
Comment (megaherz) [inline README.md:22]: .decentrd => .decentr ?

File: scripts/Dockerfile [hunk 10]
```diff
@@ -8,11 +8,12 @@ RUN make linux
 
 FROM alpine:${ALPINE_VERSION}
 RUN apk update && apk add --update ca-certificates libc6-compat
-COPY --from=build-env /go/src/github.com/Decentr-net/decentr/build/decentrd-linux-amd64 /usr/bin/decentrd
-COPY --from=build-env /go/src/github.com/Decentr-net/decentr/build/decentrcli-linux-amd64 /usr/bin/decentrcli
+COPY --from=build-env /go/src/github.com/Decentr-net/decentr/build/linux/decentrd /usr/bin/decentrd
 
 EXPOSE 26657
 EXPOSE 26656
+EXPOSE 9090
```
Comment (megaherz) [inline scripts/Dockerfile:15]: what are these new ports for?

File: README.md (hunk 7 above)
Comment (mikluke) [reply README.md:91]: Yes

File: .golangci.yml (hunk 8 above)
Comment (mikluke) [reply .golangci.yml:7]: Some of them I've commented cause of low severity

File: README.md (hunk 9 above)
Comment (mikluke) [reply README.md:22]: No, d is being be added in cosmoscmd package

File: scripts/Dockerfile (hunk 10 above)
Comment (mikluke) [reply scripts/Dockerfile:15]: According to config, but i see that for now 9090 and 9091 are redundant

**Pull request merged**